### ISSUE TITLE
Holding the key-`down` causes TerminalMenu.show returns None

### ISSUE THREAD

**Issue by minghu6** [IngoMeyer441/simple-term-menu#77]
**Env**: Ubuntu 22.04, Python 3.10, Gnome Shell

Pressing `down` too quickly causes `show` returns `None` unexpectly (especially for the long menu).

And as a workaround, If I put the `show` method into a infinite loop, it also protects from quit shortcuts. (need a specific exception to indicate a `quit` event?)

**Comment by IngoMeyer441** [comment]
Thanks for the bug report. I tried to quickly press cursor down on a long list with 100 entries, but could not get your described behavior (on Ubuntu 22.04 with gnome terminal). Do you also use the default gnome terminal?

**Comment by minghu6** [comment]
> Thanks for the bug report. I tried to quickly press cursor down on a long list with 100 entries, but could not get your described behavior (on Ubuntu 22.04 with gnome terminal). Do you also use the default gnome terminal?

Yes, i use dedault gnome shell (v42.5), sometimes it's ok, however if i rerun the script the issue occurs (just holding the key).

**Comment by minghu6** [comment]
> Thanks for the bug report. I tried to quickly press cursor down on a long list with 100 entries, but could not get your described behavior (on Ubuntu 22.04 with gnome terminal). Do you also use the default gnome terminal?

Oh. I found the detail !, I run the script use an alias which is `rlwrap python`, that's if I run python in rlwrap and then the issue occurs!

**Comment by IngoMeyer441** [comment]
Python already has built-in edit support with libreadline. Do you really need `rlwrap` here?

**Comment by minghu6** [comment]
> Python already has built-in edit support with libreadline. Do you really need `rlwrap` here?

I use rlwrap to improve Python builtin `input` behaviour : 

For an example:

A script use `input` to interact with people on terminal:
```python
input('please input:')
```
Normally, you can't edit your input which is unconvenient. 
But you could do it just using `rlwrap python yourscript.py`.

**Comment by minghu6** [comment]
> With `rlwrap simple-term-menu $(seq 1 100)` I can reproduce the behavior, you described. But I am not really sure if this is an issue with simple-term-menu or with rlwrap.

I've tried another terminal menu on a Rust impl with rlwrap, and it's alright. Maybe there are some incompatiable on terminal manipulation or comflict on libreadline?
